src/utils/events/processServices.js

```diff
@@ -22,7 +22,13 @@ const URN_PUBLISHER_REGEX = /(?=urn:ard:publisher:[a-z0-9]{16})/g
 
 module.exports = async (service, req) => {
 	// fetch prefix from configured list
-	const urnPrefix = coreIdPrefixes[service.type]
+	let urnPrefix = coreIdPrefixes[service.type]
+
+	// add a different suffix for radio text topics to not confuse subscribers with new event
+	if (req.body.event === 'de.ard.eventhub.v1.radio.text') {
+		urnPrefix = `RadioText:${urnPrefix}`
```

Comment:
this breaks the otherweise fairly lowercase spelling 
`{"id":"RadioText:urn:ard:permanent-livestream:a315d3e482f09e1b","messageId":"13323640311135945","name":"de.ard.eventhub.dev.RadioText%3Aurn%3Aard%3Apermanent-livestream%3Aa315d3e482f09e1b","status":"MESSAGE_SENT"}`

maybe `radio-text`?